src/stcal/outlier_detection/utils.py

```diff
@@ -279,6 +279,17 @@ def calc_gwcs_pixmap(in_wcs, out_wcs, in_shape):
     in_shape : list of int
         Input shape used to compute the input bounding box.
 
+    stepsize : int, optional
+        If stepsize>1, perform the full calculation on a sparser grid
+        and use interpolation to fill in the rest of the pixels.
+        Recommended if the underlying distortion correction is smooth.
+        Default 1.
+
+    order : int, optional
+        Order of the 2D spline to interpolate the sparse pixel mapping
+        if stepsize>1.  Should be either 1 (bilinear) or 3 (bicubic).
```

Comment:
```suggestion
        if ``stepsize>1``.  Supported values are: 1 (bilinear) or 3 (bicubic).
```
although, I believe, `scipy` supports orders 2 and maybe higher than 3